app/models/concerns/loggable_job.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module LoggableJob
+  extend ActiveSupport::Concern
```

Comment:
Could we also add a `rescue_from` here where we also mark the job as completed? As I understand after_perform won't run if an exception is raised during execution.

Reply:
Good point!